### PR TITLE
add code threshold for highlighting

### PR DESCRIPTION
**Pull request by lateefazeez**
## Starter Task 1
 - Add option to highlight plugin to stop running highlighting after certain code length threshold
 - Threshold currently set to 5
Screen Shot 2022-06-08 at 6 06 23 PM
Screen Shot 2022-06-08 at 6 06 59 PM

### CODE REVIEW

File: packages/lexical-code/src/index.ts [hunk 1]
```diff
@@ -716,7 +716,8 @@ function codeNodeTransform(node: CodeNode, editor: LexicalEditor) {
         const highlightNodes = getHighlightNodes(tokens);
         const diffRange = getDiffRange(node.getChildren(), highlightNodes);
         const {from, to, nodesForReplacement} = diffRange;
-        if (from !== to || nodesForReplacement.length) {
+        console.log("Length: ", code.length)
```
Comment (trueadm) [inline packages/lexical-code/src/index.ts:719]: Console.log?

File: packages/lexical-playground/src/plugins/CodeHighlightPlugin.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default function CodeHighlightPlugin(): JSX.Element | null {
   const [editor] = useLexicalComposerContext();
 
   useEffect(() => {
-    return registerCodeHighlighting(editor);
+    return registerCodeHighlighting(editor, 50);
```
Comment (fantactuka) [inline packages/lexical-playground/src/plugins/CodeHighlightPlugin.ts:17]: You can use 50 as a test value, but for a playground it should be 5000-10000

File: packages/lexical-code/src/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (lateefazeez) [reply packages/lexical-code/src/index.ts:719]: ooops! will make sure to always take them out

File: packages/lexical-playground/src/plugins/CodeHighlightPlugin.ts (hunk 2 above)
Comment (lateefazeez) [reply packages/lexical-playground/src/plugins/CodeHighlightPlugin.ts:17]: Noted

**Comment by thegreatercurve** [comment]
@lateefazeez Maybe just rebase this branch onto main again and then push it to remote? For some reason the E2E test suite doesn't to seem to have run completely.

**Pull request closed without merging**